app/src/store/views/appView.ts

```diff
@@ -28,15 +29,12 @@ export default class AppView {
     this._store = store;
   }
 
-  get fullWidth() {
-    return this._store.router.location.pathname === `${PUBLIC_URL}/pool`;
-  }
-
   /** navigate to the specified route */
   goTo(route: string) {
```

Comment:
Looks like these `goTo*` functions can all be removed now. 🙌

Reply:
I was planning to do it as part of the upgrade. But realized that refactoring navigation to use `useNavigate` and `useLocation` hooks in this PR would significantly increase the delta making it hard to manage and review.
Will refactor in a following standalone PR.

Reply:
Great idea! This definitely decreased the diff. Thanks for the clarification as well. 👍